--- src/app/_layout.tsx
+++ src/app/_layout.tsx
@@ -170,7 +170,6 @@
     "RubikGlitchPop-Regular": require("../../assets/fonts/RubikGlitchPop-Regular.ttf"),
     "Ewert-Regular": require("../../assets/fonts/Ewert-Regular.ttf"),
     "RubikDoodleShadow-Regular": require("../../assets/fonts/RubikDoodleShadow-Regular.ttf"),
-    "Cookie-Regular": require("../../assets/fonts/Cookie-Regular.ttf"),
     "Cookiemonster-gv11": require("../../assets/fonts/Cookiemonster-gv11.ttf"),
     Inter_400Regular,
     Inter_500Medium,
--- src/app/compose.tsx
+++ src/app/compose.tsx
@@ -240,7 +240,6 @@
     { id: "RubikGlitchPop-Regular", name: "GLITCH" },
     { id: "Ewert-Regular", name: "EWERT" },
     { id: "RubikDoodleShadow-Regular", name: "DOODLE" },
-    { id: "Cookie-Regular", name: "COOKIE" },
     { id: "Cookiemonster-gv11", name: "MONSTER" },
     { id: "PlayfairDisplay_400Regular", name: "PLAYFAIR" },
     { id: "SpaceGrotesk_400Regular", name: "GROTESK" },
